src/bootstrap_upgrade.sh

```diff
@@ -124,8 +124,16 @@ function upgrade() {
             return 1
         fi
 
-        if ! append_string_field "environment"; then
-            return 1
+        env_config=$(jq '.environment' $contract_config)
+        if [ "$env_config" != "null" ]; then
+            jq -e '.environment|type!="object"' $contract_config >/dev/null &&
+                echo "Contract environment invalid. Invalid format. Should be an object with string keys and string values." &&
```

Comment:
What if we say?
Contract environment config format is invalid. Its format should be an object with string keys and string values